docs/guides/publisher-client-side.md

```diff
@@ -29,12 +31,16 @@ This guide is intended for publishers with web assets who want to generate ident
 
 ## Introduction
 
-This guide outlines the [basic steps](#integration-steps) that you need to consider if you are building an integration without using an SDK. For example, you need to decide how to implement user login and logout, how to manage UID2 identity information and use it for targeted advertising, and how to refresh tokens, deal with missing identities, and handle user opt-outs. See also [FAQs](#faqs).
+This guide outlines the basic steps that you need to consider if you are building an integration without using an SDK. For example, you need to decide how to implement user login and logout, how to manage UID2 identity information and use it for targeted advertising, and how to refresh tokens, deal with missing identities, and handle user opt-outs.
+
+For a workflow diagram, see [Integration Steps](#integration-steps). See also [FAQs](#faqs).
 
 To facilitate the process of establishing client identity using UID2 and retrieving advertising tokens, the web integration steps provided in this guide rely on the UID2 SDK for JavaScript. Here's an [example application](https://example-jssdk-integ.uidapi.com/) that illustrates the integration steps described in this guide and the usage of the SDK (currently only for email addresses). For the application documentation, see [UID2 SDK Integration Example](https://github.com/IABTechLab/uid2-examples/blob/main/publisher/standard/README.md).
 
+(**GWH_LP01 any way we could go a little gentler than Danger? It looks scary. Maybe just a Note? But, your call.**)
```

Comment:
Yeah - I think something less strong would be fine, maybe a tip. The paragraph doesn't even quite make sense - it tells you that the SDK stores tokens in first-party cookies, so make sure you use the SDK for identity management (but you already are, it's the thing storing the cookie!)

Suggested change:

> The first-party cookie and local storage implementation details might change in the future. To avoid potential issues, be sure to rely on the functionality documented in the [UID2 SDK for JavaScript API Reference](../sdks/client-side-identity.md#api-reference) for your identity management.

Reply:
Awesome thx fixed.